src/Compilers/CSharp/Portable/Compiler/MethodCompiler.cs

```diff
@@ -1739,42 +1739,47 @@ private static SyntaxToken GetImplicitConstructorBodyToken(CSharpSyntaxNode cont
             }
         }
 
-        internal static BoundCall GenerateObjectConstructorInitializer(MethodSymbol constructor, DiagnosticBag diagnostics)
+        internal static BoundCall GenerateBaseConstructorInitializer(MethodSymbol constructor, DiagnosticBag diagnostics)
```

Comment:
@AlekseyTs not sure what part of it was reflecting the (parameter-less) part. The change from Generate`Object`ConstructorInitializer to Generate`Base`ConstructorInitializer is to indicate it no longer requires `System.Object` as the base parent. Are you proposing to change it to something like Generate`BaseParameterless`ConstructorInitializer?

---
In reply to: [115095529](https://github.com/dotnet/roslyn/pull/18715#discussion_r115095529) [](ancestors = 115095529)